ucm2/rockchip_es8316/HiFi.conf

```diff
@@ -0,0 +1,70 @@
+# Reference:
+# https://gitlab.manjaro.org/manjaro-arm/packages/community/pinebookpro-post-install/-/blob/4b4206c57b21044bf9c00848bd1f8b6f710d19eb/asound.state
+# https://gitlab.manjaro.org/manjaro-arm/packages/community/pinebookpro-audio/-/blob/1f567db09495acc3beb5405c9ef85c3c07b86785/audio_jack_plugged_in.sh
+
+SectionVerb {
+	Value {
+		MinBufferLevel "512"
```

Comment:
PulseAudio does not use this value. Please, remove it. I think that only the audio server from google uses this.